isis/src/qisis/objs/CubeViewport/CubeViewport.cpp

```diff
@@ -1776,7 +1776,7 @@ namespace Isis {
       e->accept();
     }
     else if ((e->key() == Qt::Key_C) &&
-             QApplication::keyboardModifiers() &&
+             QApplication::keyboardModifiers() &
              Qt::ControlModifier) {
```

Comment:
warning: enum constant in boolean context [-Wint-in-bool-context] was being thrown by using the logical &&, I replaced it with the bitwise & (only difference as I understand it is that && can short-circuit and & will not, which I don't see is a huge difference in this context). If someone see's an issue with this fix let me know

Reply:
Confirmed that functionality has not changed. Holding cntrl-c copies the filename to the clipboard in both the nightly build as well as this version.